tikv-configuration-file.md

```diff
@@ -1256,7 +1256,7 @@ Configuration items related to RocksDB
 ### `max-manifest-file-size`
 
 + The maximum size of a RocksDB Manifest file
-+ Default value: `"128MiB"`
++ Default value: `"256MiB"`
```

Comment:
```suggestion
+ Default value: `"256MiB"`. Before v9.0.0, the default value is `"128MiB"`.
```

Reply:
@qiancai We also want to change this default value in the incoming bugfix version of v7.5, v8.1 and v8.5. Thus, do we still need the "Before v9.0.0, the default value is `"128MiB"`" note?

Reply:
yes, after the code changes in other releases are done, we need to update "Before v9.0.0, the default value is `"128MiB"` to something like "Before v7.5.5, v8.1.3, 8.5.4, and v9.0.0, the default value is `"128MiB"`

Reply:
LGTM